server/mailet/mailets/pom.xml

```diff
@@ -63,6 +63,16 @@
             <artifactId>apache-james-mailbox-store</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>apache-james-rspamd</artifactId>
```

Comment:
Now we do not want the RSPAMD dependency here!

RSPAMD stuff should be in `third-party/rspamd` package.

Please relocate/cleanup everything!